pkg/scheduling/taints.go

```diff
@@ -49,7 +51,7 @@ func (ts Taints) Tolerates(pod *corev1.Pod) (errs error) {
 			tolerates = tolerates || t.ToleratesTaint(&taint)
 		}
 		if !tolerates {
-			errs = multierr.Append(errs, fmt.Errorf("did not tolerate %s=%s:%s", taint.Key, taint.Value, taint.Effect))
+			errs = multierr.Append(errs, fmt.Errorf("did not tolerate %s", pretty.Taint(ts[i])))
```

Comment:
nit: just use the taint variable that's already defined, or remove that variable and use this array index throughout

Reply:
Oh yeah. Lol